### PR TITLE
Run PHPStan via Github Action

### PR DESCRIPTION
**Pull request by rvanlaak**
see https://github.com/symfony/panther/pull/268#issuecomment-566436814

### CODE REVIEW

File: .github/workflows/static-code-analysis.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Static code analysis
+
+on: [pull_request]
+
+jobs:
+    static-code-analysis:
+        strategy:
+            matrix:
+                php: [7.1, 7.2, 7.3, 7.4]
```
Comment (stof) [inline .github/workflows/static-code-analysis.yaml]: no need to run phpstan 4 times. The result will be the same each time AFAIK.

Comment (rvanlaak) [reply .github/workflows/static-code-analysis.yaml]: I actually did have additional errors while running PHPStan with the PHP7.4 image.

@jakzal did mention changing the image to be running `composer install` in https://github.com/php-translation/symfony-storage/pull/38#discussion_r356660383

Will revert the matrix for now 👍

Comment (stof) [reply .github/workflows/static-code-analysis.yaml]: Were the different images using the same version of PHPStan ?

File: phpstan.neon [hunk 2]
```diff
@@ -1,4 +1,8 @@
 parameters:
+	level: 6
+    paths:
```
Comment (stof) [inline phpstan.neon]: indentation looks wrong here

Comment (rvanlaak) [reply phpstan.neon]: Thanks. Fixed

**Comment by dunglas** [comment]
Closed by #372. Thanks for your contribution!

**Pull request closed without merging**